src/Turbo/src/Bridge/Mercure/TurboStreamListenRenderer.php

```diff
@@ -55,6 +63,29 @@ public function renderTurboStreamListen(Environment $env, $topic): string
             $controllerAttributes['topic'] = current($topics);
         }
 
+        if (isset($eventSourceOptions)) {
+            if (
+                null !== $this->authorization
+                && null !== $this->requestStack
+                && (isset($eventSourceOptions['subscribe']) || isset($eventSourceOptions['publish']) || isset($eventSourceOptions['additionalClaims']))
+                && null !== $request = $this->requestStack->getMainRequest()
+            ) {
+                $this->authorization->setCookie(
+                    $request,
+                    $eventSourceOptions['subscribe'] ?? [],
+                    $eventSourceOptions['publish'] ?? [],
+                    $eventSourceOptions['additionalClaims'] ?? [],
+                    $eventSourceOptions['transport'] ?? null,
+                );
```

Comment:
This will throw if the method is called twice during the render of the page.

What are the goal of the various options here ? 

Or, say differently, why not use the mercure extension here ?  (real question i'm not sure to get)

Reply:
Do you mean the Mercure Twig extension?
If so, it doesn't seem to work 😅. The dependency isn't being resolved, and to be honest, I'm not sure why.